apple/macbook-pro/3-1/README.wiki

```diff
@@ -0,0 +1,5 @@
+= Apple MacBook Pro 3,1 =
```

Comment:
Can you also add this to the README and the flake.nix as described in https://github.com/NixOS/nixos-hardware/blob/master/CONTRIBUTING.md#2-adding-entry ?